src/Agent/NewRelic/Agent/Core/DataTransport/ConnectionHandler.cs

```diff
@@ -276,7 +276,7 @@ private void LogConfigurationMessages(ServerConfiguration serverConfiguration)
                 if (string.IsNullOrEmpty(message.Text))
                     continue;
 
-                var logMethod = ServerLogLevelMap.GetValueOrDefault(message.Level) ?? Log.Info;
+                var logMethod = ServerLogLevelMap.GetValueOrDefault(message.Level) ?? new Action<string>((s) => Log.Info(s));
```

Comment:
Is there a way to avoid allocating an action instance and a lambda here?

Reply:
Done